dags/pipeline_small.py

```diff
@@ -141,30 +157,34 @@ def remove_unblurred_images(**context):
         volume_mounts=[],
     )
 
-    blur_images = KubernetesPodOperator(
-        task_id='blur_images',
-        namespace=AKS_NAMESPACE,
-        image=BLUR_CONTAINER_IMAGE,
-        env_vars=get_generic_vars(),
-        cmds=["python"],
-        arguments=["/app/detect.py",
-                   "--date", DATE],
-        labels=DAG_LABEL,
-        name=DAG_ID,
-        image_pull_policy="Always",
-        get_logs=True,
-        in_cluster=True,
-        is_delete_operator_pod=True, 
-        log_events_on_failure=True, 
-        hostnetwork=True, 
-        reattach_on_restart=True,
-        dag=dag,
-        startup_timeout_seconds=3600,
-        execution_timeout=timedelta(hours=4),
-        node_selector={"nodetype": AKS_NODE_POOL},
-        volumes=[],
-        volume_mounts=[],
-    )
+    blur_images_list = []
+    for worker_id in num_workers:
+        blur_images = KubernetesPodOperator(
+            task_id='blur_images',
+            namespace=AKS_NAMESPACE,
+            image=BLUR_CONTAINER_IMAGE,
+            env_vars=get_generic_vars(),
+            cmds=["python"],
+            arguments=["/app/detect.py",
+                       "--date", DATE,
+                       "--worker-id", worker_id],
+            labels=DAG_LABEL,
+            name=DAG_ID,
```

Comment:
Combination with batched:
- get a list with filenames that still need be processed
- Split the list of filenames over n workers